products/opensuse/main.pm

```diff
@@ -654,19 +665,50 @@ sub load_extra_tests () {
     return 0;
 }
 
+sub load_otherDE_tests() {
```

Comment:
as load_otherDE_tests is not passed to load_test, this function name
does not become visible in the openQA webui...
the only one that IS visible is install_ohterDE_pattern - which is
shows like this in my instance - let's discuss this on IRC and then
post our agreed final solution to this topic here
On Fri, 2016-06-10 at 06:13 -0700, Oliver Kurz wrote:
> 
> I am not talking about the testsuite but the test modules name, e.g.
> the rows with the colored background on the details page.
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 
>